diff -Naur orig/src-qt5/core/lumina-desktop/lumina-desktop.pro patched/src-qt5/core/lumina-desktop/lumina-desktop.pro
--- orig/src-qt5/core/lumina-desktop/lumina-desktop.pro	2016-10-21 16:48:55.000000000 +0200
+++ patched/src-qt5/core/lumina-desktop/lumina-desktop.pro	2016-10-23 20:04:47.614916035 +0200
@@ -78,7 +78,7 @@
 			wallpapers/Lumina_Wispy_blue-grey-zoom.jpg \
 			wallpapers/Lumina_Wispy_grey-blue.jpg \
 			wallpapers/Lumina_Wispy_grey-blue-zoom.jpg 
-wallpapers.path = $${L_SHAREDIR}/wallpapers/Lumina-DE
+wallpapers.path = $${L_SHAREDIR}/backgrounds/Lumina-DE
 
 
 defaults.files = defaults/luminaDesktop.conf \
diff -Naur orig/src-qt5/core-utils/lumina-config/pages/page_wallpaper.cpp patched/src-qt5/core-utils/lumina-config/pages/page_wallpaper.cpp
--- orig/src-qt5/core-utils/lumina-config/pages/page_wallpaper.cpp	2016-10-21 16:48:55.000000000 +0200
+++ patched/src-qt5/core-utils/lumina-config/pages/page_wallpaper.cpp	2016-10-23 20:04:47.614916035 +0200
@@ -195,7 +195,7 @@
 
 void page_wallpaper::deskbgadded(){
   //Prompt the user to find an image file to use for a background
-  QString dir = LOS::LuminaShare().section("/lumina-desktop",0,0)+"/wallpapers/Lumina-DE";
+  QString dir = LOS::LuminaShare().section("/lumina-desktop",0,0)+"/backgrounds/Lumina-DE";
   qDebug() << "Looking for wallpaper dir:" << dir;
   if( !QFile::exists(dir) ){ dir = QDir::homePath(); }
   QStringList imgs = LUtils::imageExtensions();
@@ -228,7 +228,7 @@
 
 void page_wallpaper::deskbgdiradded(){
   //Add the files from a single directory
- QString dir = LOS::LuminaShare().section("/lumina-desktop",0,0)+"/wallpapers/Lumina-DE";
+ QString dir = LOS::LuminaShare().section("/lumina-desktop",0,0)+"/backgrounds/Lumina-DE";
   qDebug() << "Looking for wallpaper dir:" << dir;
   if( !QFile::exists(dir) ){ dir = QDir::homePath(); }
   dir = QFileDialog::getExistingDirectory(this, tr("Find Background Image Directory"), dir, QFileDialog::ReadOnly);
@@ -253,7 +253,7 @@
 
 void page_wallpaper::deskbgdirradded(){
   //Recursively add files from a directory
- QString dir = LOS::LuminaShare().section("/lumina-desktop",0,0)+"/wallpapers/Lumina-DE";
+ QString dir = LOS::LuminaShare().section("/lumina-desktop",0,0)+"/backgrounds/Lumina-DE";
   qDebug() << "Looking for wallpaper dir:" << dir;
   if( !QFile::exists(dir) ){ dir = QDir::homePath(); }
   dir = QFileDialog::getExistingDirectory(this, tr("Find Background Image Directory"), dir, QFileDialog::ReadOnly);
